src/main/java/taskat2/Human.java

```diff
@@ -0,0 +1,4 @@
+package taskat2;
+
+public class Human {
+}
```

Comment:
what is going on with classes and packages inside AT-4?
this comment belongs as well to all classes in taskat2 and taskat3 packages. What are they doing here?

Reply:
removed them